### ISSUE TITLE
[Tracker]: Upgrade react-router-dom to latest version

### ISSUE THREAD

**Issue by anshgoyalevil** [jaegertracing/jaeger-ui#1825]
### What happened?

.

### Steps to reproduce

.

### Expected behavior

.

**Comment by anshgoyalevil** [comment]
Yes, we would have to upgrade it sequentially version by version, keeping the project intact

**Comment by priyanshu-kun** [comment]
@anshgoyalevil  do we have to migrate it to v5 before v6?

**Comment by Piyushrathoree** [comment]
@yurishkuro is this still unresolved ? if not I would love to work on it.

**Comment by parshipcy** [comment]
done #3313
